samza-yarn/src/main/scala/org/apache/samza/job/yarn/ClientHelper.scala

```diff
@@ -191,12 +193,15 @@ class ClientHelper(conf: Configuration) extends Logging {
       }
     }
 
-    if (UserGroupInformation.isSecurityEnabled()) {
+    if (UserGroupInformation.isSecurityEnabled) {
       validateJobConfig(config)
 
       setupSecurityToken(fs, containerCtx)
       info("set security token for %s" format appId.get)
 
+      val securityManager = new SamzaContainerSecurityManager(config, new YarnConfiguration())
+      securityManager.setApplicationAcl(containerCtx)
```

Comment:
Would prefer to inline the `setApplicationAcl` here (or inside `setupSecurityTokens`) instead of exposing it as a method on the `SecurityManager`. A nice property is that all mutations to the "context" object happen in a single place. 

Alternately, If you want to move this to `SecurityManager`, we could move other aspects as well, including token setup and renewal. Currently the security-manager does only renewal.

Reply:
ClientHelper.scala and YarnClusterResourceManager.java share the setApplicationAcl logic here. If you prefer this small piece of duplicate codes instead of a common place, I guess that's fine. What do you think?